spec/build-spec.js

```diff
@@ -35,6 +37,8 @@ describe('Build', () => {
         return specHelpers.vouch(fs.realpath, dir);
       }).then( (dir) => {
         directory = dir + '/';
+        const createdHomeDir = temp.mkdirSync('atom-build-spec-home');
+        os.homedir = () => createdHomeDir;
```

Comment:
only in this spec is using async, in all the others specs is being done sync way.

is it really necessary for a test?